### PR TITLE
Update doc-validator

### PR DESCRIPTION
**Pull request by jdbaldry**
https://github.com/grafana/technical-documentation/blob/main/tools/cmd/doc-validator/CHANGELOG.md contains the full list of changes.

You can see the failures in https://github.com/grafana/mimir/actions/runs/4895741370/jobs/8741701493?pr=4943.

### CODE REVIEW

File: docs/docs.mk [hunk 1]
```diff
@@ -91,10 +98,10 @@ docs-no-pull: make-docs
 .PHONY: docs-debug
 docs-debug: ## Run Hugo web server with debugging enabled. TODO: support all SERVER_FLAGS defined in website Makefile.
 docs-debug: make-docs
-	WEBSITE_EXEC='hugo server --debug' $(PWD)/make-docs $(PROJECTS)
+	WEBSITE_EXEC='hugo server --bind 0.0.0.0 --port 3002 --debug' $(PWD)/make-docs $(PROJECTS)
 
 .PHONY: doc-validator
-doc-validator: ## Run docs-validator on the entire docs folder.
+doc-validator: ## Run doc-validator on the entire docs folder.
```
Comment (osg-grafana) [inline docs/docs.mk:104]: Nice catch!

File: docs/make-docs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
-# Source of this file is https://github.com/grafana/technical-documentation/scripts/make-docs.
+# The source of this file is https://raw.githubusercontent.com/grafana/writers-toolkit/main/docs/make-docs.
```
Comment (osg-grafana) [inline docs/make-docs:2]: Love this self-documentation. :)

**Review by osg-grafana — APPROVED** [review]
Unblocking with the assumption that another (engineering) maintainer will have a look and over another approval.

**Comment by pracucci** [comment]
Hi @jdbaldry. Out of curiosity, what's the state of this PR?

**Comment by jdbaldry** [comment]
Closing in favor of #5172

**Pull request closed without merging**